File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/SykefraværsstatistikkVirksomhetUtenVarighet.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.importering;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
// TODO: rename SykefraværsstatistikkVirksomhet til SykefraværsstatistikkVirksomhetMedVarighet og denne til SykefraværsstatistikkVirksomhet
public class SykefraværsstatistikkVirksomhetUtenVarighet implements Sykefraværsstatistikk {
    private int årstall;
    private int kvartal;
    private String orgnr;
    private int antallPersoner;
    private BigDecimal tapteDagsverk;
    private BigDecimal muligeDagsverk;
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/fellesdomene/SykefraværsstatistikkVirksomhetUtenVarighet.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene

import java.math.BigDecimal

data class SykefraværsstatistikkVirksomhetUtenVarighet(
    override val årstall: Int = 0,
    override val kvartal: Int = 0,
    val orgnr: String,
    override val antallPersoner: Int = 0,
    override val tapteDagsverk: BigDecimal,
    override val muligeDagsverk: BigDecimal,
) : Sykefraværsstatistikk

